emu/src/internal_memory.rs

```diff
@@ -3,6 +3,9 @@ use crate::io_device::IoDevice;
 use crate::io_registers::LCDRegisters;
 
 pub struct InternalMemory {
+    /// From 0x00000000 to 0x00003FFF (16 KBytes)
+    bios_system_rom: [u8; 0x3FFF],
```

Comment:
The size is wrong. The right one is `0x4000`. I open a issue about this #97.

Reply:
Done :)